providers/fips/fipsprov.c

```diff
@@ -138,7 +141,7 @@ static int fips_get_params_from_core(FIPS_GLOBAL *fgbl)
     * OSSL_PROV_FIPS_PARAM_SECURITY_CHECKS and
     * OSSL_PROV_FIPS_PARAM_TLS1_PRF_EMS_CHECK are not self test parameters.
     */
-    OSSL_PARAM core_params[10], *p = core_params;
+    OSSL_PARAM core_params[12], *p = core_params;
```

Comment:
This is one of the points where merging may go really bad with all the PR's

Reply:
That's why I bumped it by two -- there is a second PR that wants to modify this.
Maybe set it to something large with an assert if it overflows.